src/App.tsx

```diff
@@ -1,7 +1,37 @@
 import React from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  key: boolean,
+};
```

Comment:
your key variable have to be string or null, when you initialize it set it to null then change it to result of handleClick function (event.key), right now your solution doesnt work as intended